recipe/meta.yaml

```diff
@@ -12,16 +12,17 @@ source:
 build:
   number: 0
   script: python setup.py install --offline --no-git --single-version-externally-managed --record record.txt
-
+  skip: true  # [py27]
```

Comment:
might be more accurate to say `[py<3.6]`??

Reply:
it seems to already default to that? the re-render has not added any 3.5 builds.

Reply:
although it will have the same effect because 3.6 is the min conda-forge version